packages/@coorpacademy-components/src/molecule/skills-chart-side-information-item/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import style from './style.css';
+
+const SkillsChartSideInformationItem = props => {
+  const {title, value, legend, icon: iconProps} = props;
+
+  return (
+    <div className={style.InformationElement}>
```

Comment:
we really need this level of div. You can't merge `InformationElement` and `content`.